### PR TITLE
Add linting to the CI

### LINKED ISSUE #4: Add markdown linting
Currently we have no linting on our markdown files - would be good to add some and make the formatting more consistent.

### PR DESCRIPTION
**Pull request by hug-dev**
This commit lints all the file with the vmdfmt tool. It also adds this
tool to the CI in a workflow that checks that the book builds and that
the files are well formatted.

Close #4 

Signed-off-by: Hugues de Valon <hugues.devalon@arm.com>

### CODE REVIEW

**Comment by hug-dev** [comment]
I am considering installing `cmdfmt` inside a Docker container and execute it there. I can't figure how to install it inside the GitHub Actions 💀

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,23 +1,28 @@
-name: Build Github mdBook
+name: Continuous Integration
 
-on:
-  push:
-    branches:
-    - master
+on: [push, pull_request]
```
Comment (ionut-arm) [inline .github/workflows/ci.yml]: Are you sure you want to build the book on every PR?

File: src/README.md [hunk 2]
```diff
@@ -1,38 +1,24 @@
...
+to the contribution guidelines.
 
 # License
 
-Parsec and all the repositories under the `parallaxsecond` organization are provided under Apache-2.0.
-Contributions to this project are accepted under the same license.
+Parsec and all the repositories under the `parallaxsecond` organization are provided under
+Apache-2.0. Contributions to this project are accepted under the same license.
+
+*Copyright (c) 2019, Arm Limited. All rights reserved.*
```
Comment (ionut-arm) [inline src/README.md]: Shouldn't this be 2020 since you now changed the file?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ionut-arm) [reply .github/workflows/ci.yml]: Ah, I just saw the `Deploy` workflow!

File: src/README.md (hunk 2 above)
Comment (hug-dev) [reply src/README.md]: We are currently discussing ways of simplifying the copyright notice and will update it to 2019-2020 depending on how the discussion goes.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -1,23 +1,29 @@
...
+  format:
+    name: Check formatting
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Create a Go workspace and install vmdfmt
+      run: |
+          mkdir /tmp/go
+          export GOPATH=/tmp/go
+          go get github.com/bobertlo/vmd/cmd/vmdfmt
```
Comment (hug-dev) [inline .github/workflows/ci.yml]: Hack 101 🔨

Comment (ionut-arm) [reply .github/workflows/ci.yml]: _\<hacker voice\>_ I'm in

Comment (hug-dev) [reply .github/workflows/ci.yml]: `fsociety`

**Pull request merged**